### ISSUE TITLE
how to use batch norm on onnx model?

### ISSUE THREAD

**Issue by zxgm** [onnx/onnx#2475]
how to use batch norm on onnx model?

**Comment by daquexian** [comment]
Could you elaborate, please? ONNX has [BatchNormalization](https://github.com/onnx/onnx/blob/master/docs/Operators.md#BatchNormalization) op.
